plugin/src/main/java/io/snyk/eclipse/plugin/domain/ProductConstants.java

```diff
@@ -0,0 +1,8 @@
+package io.snyk.eclipse.plugin.domain;
+
+public final class ProductConstants {
```

Comment:
Why not an interface? Because of immutability, they are a bit preferred in Javaland for constants.